pkg/reconciler/taskrun/resources/apply.go

```diff
@@ -108,24 +109,37 @@ func ApplyContexts(spec *v1beta1.TaskSpec, rtr *ResolvedTaskResources, tr *v1bet
 	return ApplyReplacements(spec, replacements, map[string][]string{})
 }
 
-// ApplyWorkspaces applies the substitution from paths that the workspaces in w are mounted to, the
-// volumes that wb are realized with in the task spec ts and the PersistentVolumeClaim names for the
+// ApplyWorkspaces applies the substitution from paths that the workspaces in decls mounted to, the
+// volumes that binds are realized with in the task spec and the PersistentVolumeClaim names for the
 // workspaces.
-func ApplyWorkspaces(spec *v1beta1.TaskSpec, w []v1beta1.WorkspaceDeclaration, wb []v1beta1.WorkspaceBinding) *v1beta1.TaskSpec {
+func ApplyWorkspaces(spec *v1beta1.TaskSpec, decls []v1beta1.WorkspaceDeclaration, binds []v1beta1.WorkspaceBinding) *v1beta1.TaskSpec {
```

Comment:
`decls` for declared workspaces ?

Reply:
yeah, exactly right!  Maybe `declarations` to make it a bit clearer?

Reply:
@sbwsg maybe :stuck_out_tongue_closed_eyes: it's a nit, but I would indeed prefer `declarations` :stuck_out_tongue_closed_eyes:

Reply:
updated!